.github/workflows/scripts/setup_bundler

```diff
@@ -56,12 +51,10 @@ function install_desired_bundler_version {
   echo "DEBUG: running 'gem list bundler'"
   gem list bundler
   echo "DEBUG: RUBY_VERSION = $RUBY_VERSION"
-  if [[ $RUBY_VERSION != "2.2.10" ]]; then 
-    echo "DEBUG: running 'gem uninstall bundler'"
-    gem uninstall bundler
-    echo "DEBUG: running 'gem list bundler' again"
-    gem list bundler
-  fi
+  echo "DEBUG: running 'gem uninstall bundler'"
+  gem uninstall bundler
+  echo "DEBUG: running 'gem list bundler' again"
+  gem list bundler
 }
 
 function configure_bundler {
```

Comment:
This `configure_bundler` and its lone caller should be safe to remove as part of this PR.

Reply:
Addressed [here](https://github.com/newrelic/newrelic-ruby-agent/pull/1778/commits/f095cc4f9ce638312a084ebd8bb7822fc7f2c3f2).